source/ceylon/time/internal/TimeOfDay.ceylon

```diff
@@ -152,12 +152,9 @@ shared class TimeOfDay(millisecondsOfDay) satisfies Time {
         return time(hours, minutes, seconds, milliseconds);
     }
 
-    "Return _true_ if it have same type and milliseconds of day."
+    "Two `Time`s are considered equals if they the same _milliseconds of day_."
     shared actual Boolean equals( Object other ) {
-        if ( is TimeOfDay other ) {
-            return millisecondsOfDay == other.millisecondsOfDay;
-        }
-        return false;
+        return (super of Time).equals(other);
```

Comment:
Without this refining i have: `May not inherit two declarations with the same name unless redefined in subclass: equals is defined by supertypes Identifiable and Time`